Makefile

```diff
@@ -103,6 +102,6 @@ set_env_class_prod:
 vagrant-deploy:
 	vagrant/deploy.sh
 
-.PHONY: deploy_pipelines
-deploy_pipelines:
+.PHONY: pipelines
+pipelines: ## Upload pipelines to Concourse
```

Comment:
I wonder whether we should have a guard here to prevent this being run on its own.

Perhaps we should rename the existing `check-env-vars` target to `check-deploy-env`, and then add a new `check-env-vars` that checks for the variables these jobs require. It's probably enough to just check for one of the vars set by the env targets (eg `MAKEFILE_ENV_TARGET`).